site/realm/data_sources/mongodb-atlas/aiidprod/incidents/schema.json

```diff
@@ -44,7 +44,39 @@
         },
         "title": {
             "bsonType": "string"
-        }
+        },
+        "nlp_similar_incidents": {
+          "bsonType": "array",
+          "items": {
+              "bsonType": "object",
+              "properties": {
+                  "incident_id": {
+                      "bsonType": "int"
```

Comment:
In the future (it doesn't seem to be necessary $now), we might want to set up a Realm Relationship so related incidents can be queried as:

```
nlp_similar_incidnets {
 incident {
   incident_id
   title
 }
}
```

Because as it is, if someone wants to display a list of related incidents, he will have first to query `incident_id`s and then query again to get the titles.

Similar to https://github.com/responsible-ai-collaborative/aiid/blob/master/site/realm/data_sources/mongodb-atlas/aiidprod/incidents/relationships.json

Reply:
I agree.